var/spack/repos/builtin/packages/py-slepc4py/package.py

```diff
@@ -11,13 +11,15 @@ class PySlepc4py(PythonPackage):
     """
 
     homepage = "https://gitlab.com/slepc/slepc4py"
-    url      = "https://slepc.upv.es/download/distrib/slepc4py-3.16.2.tar.gz"
+    url      = "https://slepc.upv.es/download/distrib/slepc4py-3.17.1.tar.gz"
     git      = "https://gitlab.com/slepc/slepc.git"
 
     maintainers = ['joseeroman', 'balay']
 
     version('main', branch='main')
+    version('3.17.1', sha256='967d5d045526088ff5b7b2cde76f8b4d1fee3a2a68481f85224b0795e6613eb9 ')
```

Comment:
```suggestion
    version('3.17.1', sha256='967d5d045526088ff5b7b2cde76f8b4d1fee3a2a68481f85224b0795e6613eb9')
```

Reply:
This fix is required (to resolve unit test failure).